plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/ConfigurationCacheTest.kt

```diff
@@ -58,15 +59,13 @@ class ConfigurationCacheTest : AbstractPluginTest() {
             val formatTaskName = KtLintFormatTask.buildTaskNameForSourceSet("main")
             build(
                 configurationCacheFlag,
-                configurationCacheWarnFlag,
```

Comment:
Can you elaborate on why this isn't needed anymore?

Reply:
I removed it everywhere in this test now. Reason being that this would not fail the test when configuration actually fails. With relative path you would get a message that reads something like
```
FAILURE: Build failed with an exception.

* What went wrong:
Configuration cache problems found in this build.

2 problems were found storing the configuration cache, 1 of which seems unique.
```
So we probably want this test to fail in the future if someone implements something that is not capable of configuration caching?